mozregression/main.py

```diff
@@ -69,18 +69,11 @@ def parse_args(argv=None):
                         help=("last known good nightly build. This option is"
                               " incompatible with --good."))
 
-    parser.add_argument("--inbound",
-                        action="store_true",
-                        help=("use inbound instead of nightlies (use"
-                              " --good-rev and --bad-rev options."))
-
     parser.add_argument("--bad-rev", dest="first_bad_revision",
-                        help=("first known bad revision (use with"
-                              " --inbound)."))
+                        help=("first known bad revision"))
```

Comment:
Could you add something to indicate that we bisect inbound ?
Maybe "first know bad revision (for inbound bisection)"
Also, please remove the parentheses around the help text if it is a one line.